### PR TITLE
Added new version of OpenSSL and LibreSSL, as well as missing Gzip st…

### PR DESCRIPTION
**Pull request by filip-minic**
…atic module

### CODE REVIEW

File: nginx-autoinstall.sh [hunk 1]
```diff
@@ -8,8 +8,8 @@ fi
 # Variables
 NGINX_MAINLINE_VER=1.15.8
 NGINX_STABLE_VER=1.14.2
-LIBRESSL_VER=2.7.5
-OPENSSL_VER=1.1.1
+LIBRESSL_VER=2.8.3
```
Comment (skid9000) [inline nginx-autoinstall.sh]: LibreSSL 2.9.0 is out : https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.9.0-relnotes.txt

File: nginx-autoinstall.sh [hunk 2]
```diff
@@ -8,8 +8,8 @@ fi
...
+OPENSSL_VER=1.1.1a
```
Comment (skid9000) [inline nginx-autoinstall.sh]: OpenSSL 1.1.1b is out : https://www.openssl.org/source/openssl-1.1.1b.tar.gz

File: conf/nginx.conf [hunk 3]
```diff
@@ -26,6 +26,7 @@ http {
     gzip_comp_level 6;
     gzip_proxied any;
     gzip_types *;
+    gzip_static on;
```
Comment (angristan) [inline conf/nginx.conf]: Please make a separate PR for this

File: nginx-autoinstall.sh (hunk 1 above)
Comment (filip-minic) [reply nginx-autoinstall.sh]: Thank you for the suggestion, however this is just a development release.

Comment (Luclu7) [reply nginx-autoinstall.sh]: LibreSSL 2.9.2 is now ready for production usage https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.9.2-relnotes.txt

**Comment by skid9000** [comment]
I think we can close this one since there is #90 now (and the submitter seems inactive) 🤔
